src/app/cube/details/components/action-panel/action-panel.component.ts

```diff
@@ -110,7 +110,9 @@ export class ActionPanelComponent implements OnInit, OnDestroy {
   }
 
   get mapAndTag() {
-    if (this.auth.user && this.auth.user.accessGroups && !this.userIsAuthor) {
+    const untaggable = this.learningObject.status === LearningObject.Status.RELEASED
+                              || this.learningObject.status === LearningObject.Status.UNRELEASED;
+    if (this.auth.user && this.auth.user.accessGroups && !this.userIsAuthor && !untaggable) {
       const privileges = ['admin', 'editor', 'mapper'];
```

Comment:
the diff doesn't show this but this line was changed to remove curators and reviewers from the list.